pkg/testutil/api_check.go

```diff
@@ -0,0 +1,123 @@
+// Copyright 2022 TiKV Project Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package testutil
+
+import (
+	"encoding/json"
+	"io"
+	"net/http"
+	"strings"
+
+	"github.com/pingcap/check"
+	"github.com/tikv/pd/pkg/apiutil"
+)
+
+// Status is used to check whether http response code is equal given code
+func Status(c *check.C, code int) func([]byte, int) {
+	return func(_ []byte, i int) {
+		c.Assert(i, check.Equals, code)
+	}
+}
+
+// StatusOK is used to check whether http response code is equal http.StatusOK
+func StatusOK(c *check.C) func([]byte, int) {
+	return Status(c, http.StatusOK)
+}
+
+// StatusNotOK is used to check whether http response code is not equal http.StatusOK
+func StatusNotOK(c *check.C) func([]byte, int) {
+	return func(_ []byte, i int) {
+		c.Assert(i == http.StatusOK, check.IsFalse)
+	}
+}
```

Comment:
Is it duplicated with `Status`?

Reply:
> Is it duplicated with `Status`?

Easy to use, so the user doesn't have to add code `200` every time.